config/notifiers.go

```diff
@@ -220,6 +220,34 @@ func (c *PagerdutyConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	return nil
 }
 
+// SlackAction configures a single Slack action that is sent with each notification.
+// Each action must contain a type, text, and url.
+// See https://api.slack.com/docs/message-attachments#actions for more information.
```

Comment:
The correct link is `https://api.slack.com/docs/message-attachments#action_fields`.